Source/Caches/HACache.swift

```diff
@@ -355,6 +385,40 @@ public class HACache<ValueType> {
         })
     }
 
+    /// Do the underlying subscribe with optional population
+    /// - Parameters:
+    ///   - subscription: The subscription info
+    ///   - connection: The connection to subscribe on
+    ///   - populate: The populate request, for re-issuing when needed (optional)
+    ///   - cache: The cache whose state should be updated
+    /// - Returns: The cancellable token for the subscription
+    private static func startSubscribe<ValueType>(
+        to subscription: HACacheSubscribeInfo<ValueType?>,
+        on connection: HAConnection,
+        populate: HACachePopulateInfo<ValueType>?,
```

Comment:
Maybe dropping this parameter would make this cleaner, since you won't have missing cases below and you can just noop reissuePopulate entirely.